Dockerfile

```diff
@@ -1,17 +1,23 @@
 FROM ubuntu:xenial
-MAINTAINER Fmstrat <fmstrat@NOSPAM.NO>
+
+LABEL maintainer="Fmstrat <fmstrat@NOSPAM.NO>"
 
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt-get update && apt-get upgrade -y
+COPY init.sh /init.sh
```

Comment:
Please relocate the COPY line to the bottom so that builds can take advantage of the cache. Thanks!